mathoxide-lib/src/views.rs

```diff
@@ -50,14 +80,23 @@ mod test {
     use super::*;
 
     #[test]
-    fn simple_view_check_last_item() {
-        let view = SimpleView::new([2, 3, 4]);
-        assert_eq!(view.translate([1, 2, 3]), view.size() - 1);
+    fn contiguous_view_check() {
+        assert_eq!(ContiguousView::new([2, 3]).is_contiguous(), true);
+    }
+
+    #[test]
+    fn contiguous_view_check_last_item() {
+        let view = ContiguousView::new([2, 3, 4]);
+        assert_eq!(view.translate([1, 2, 3]), view.numel() - 1);
+
+        let offset: usize = 5;
+        let view = ContiguousView::new_with_offset([2, 3, 4], offset);
+        assert_eq!(view.translate([1, 2, 3]), view.numel() - 1 + offset);
```

Comment:
We could move these lines in a separate test case. They test the same thing but they are logically independent of the first assertion.